programmers2/자릿수-더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+    const N= (n + "")
```

Comment:
아래와 같이 코드를 봤을때 문자열로 바뀐다를 알수있는 코드가 더 좋습니다.
 const N = n.toString() or const N = String(n)
```suggestion
   const N = n.toString()
```